opentelemetry/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [v0.16.0](https://github.com/open-telemetry/opentelemetry-rust/compare/v0.15.0...v0.16.0)
+
+### Changed
+
+- Renamed `label` to `attribute` to align with otel specification #591
```

Comment:
nit: we usually use the PR id here

Reply:
Pushed a fix  with the updated PR # on the changelog.